apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1707,4 +1707,297 @@ describe('[Rooms]', function () {
 				.end(done);
 		});
 	});
+	describe('/rooms.isMember', async () => {
+		let testChannel;
+		let testGroup;
+		let testDM;
+
+		const fakeRoomId = `room.test.${Date.now()}-${Math.random()}`;
+		const fakeUserId = `user.test.${Date.now()}-${Math.random()}`;
+
+		const testChannelName = `channel.test.${Date.now()}-${Math.random()}`;
+		const testGroupName = `group.test.${Date.now()}-${Math.random()}`;
+
+		let testUser1;
+		let testUser2;
+		let testUserNonMember;
+		let testUser1Credentials;
+		let testUserNonMemberCredentials;
+
+		it('create users', async () => {
+			[testUser1, testUser2, testUserNonMember] = await Promise.all([createUser(), createUser(), createUser()]);
+			[testUser1Credentials, testUserNonMemberCredentials] = await Promise.all([
+				login(testUser1.username, password),
+				login(testUserNonMember.username, password),
+			]);
+		});
+
+		it('create a channel', (done) => {
+			createRoom({
+				type: 'c',
+				name: testChannelName,
+				members: [testUser1.username, testUser2.username],
+			}).end((err, res) => {
+				testChannel = res.body.channel;
+				done();
+			});
+		});
+		it('create a group', (done) => {
+			createRoom({
+				type: 'p',
+				name: testGroupName,
+				members: [testUser1.username, testUser2.username],
+			}).end((err, res) => {
+				testGroup = res.body.group;
+				done();
+			});
+		});
+		it('create a direct message room', (done) => {
+			createRoom({
+				type: 'd',
+				username: testUser2.username,
+				credentials: testUser1Credentials,
+			}).end((err, res) => {
+				testDM = res.body.room;
+				done();
+			});
+		});
```

Comment:
Since we don't have any assertion here and it's just a setup step, I think this would fit better by using a `before` instead of `it`, also, we should use `await` here instead of `done()` as you did in the user's creation.